image/Makefile

```diff
@@ -75,8 +76,8 @@ engine-$(ARCH)-docker-compat.tar: image-linux
 
 image-linux-dm: $(ENGINE_DIR)/Dockerfile.engine-dm
 	$(IMAGE_BUILD)
-	docker tag $(IMAGE_WITH_TAG) $(IMAGE_WITH_TAG)-dm
-	echo $(IMAGE_WITH_TAG)-dm > $@
+	docker tag $(IMAGE_WITH_ORG):$(STATIC_VERSION) $(IMAGE_WITH_ORG)-dm:$(STATIC_VERSION)
+	echo $(IMAGE_WITH_ORG)-dm:$(STATIC_VERSION) > $@
```

Comment:
This wont work out with the PR I posted to your `f` branch.

I think if you use the pattern:
```
image-linux-dm: ENGINE_IMAGE:=$(ENGINE_IMAGE)-dm
image-linux-dm: $(ENGINE_DIR)/Dockerfile.engine-dm
        $(IMAGE_BUILD)
        ...
```

that might work.

Reply:
changed